src/core/blockchain/types/client.ts

```diff
@@ -1,14 +1,25 @@
 import BigNumber from 'bignumber.js';
 import { RpcClient } from '../../utils/rpc-client';
 import { IBlockchainNetwork, ChainIdType } from './network';
-import { IFeeOptions } from './transaction';
+import { IFeeOptions, IBlockchainTransaction } from './transaction';
 import { IBlockchainNameService } from '.';
 
 export interface IBlockInfo {
     number: number;
     hash?: string;
 }
 
+export interface ITransactionReceipt {
```

Comment:
i didn't saw this interface being used, if it's used move it under ethereum folder as it's ethereum specific

Reply:
removed it now